src/tcgwars/logic/impl/gen3/UnseenForces.groovy

```diff
@@ -795,18 +795,27 @@ public enum UnseenForces implements LogicCardInfo {
         weakness F
         pokeBody "Intimidating Ring", {
           text "As long as Ursaring is your Active Pokémon, your opponent's Basic Pokémon can't attack or use any Poké-Powers."
-          delayedA {
-            before CHECK_ATTACK_REQUIREMENTS, {
-              if (self.active && ef.attacker.owner != self.owner && !ef.attacker.evolution) {
-                wcu "Intimidating Ring prevents attack"
-                prevent()
+          def eff1, eff2
+          onActivate {
+            eff1 = delayed {
+              before CHECK_ATTACK_REQUIREMENTS, {
+                if (self.active && ef.attacker.owner != self.owner && !ef.attacker.evolution) {
+                  wcu "Intimidating Ring prevents attack"
+                  prevent()
+                }
               }
             }
-          }
-          getterA (IS_ABILITY_BLOCKED) { Holder h->
-            if (self.active && h.effect.target.owner != self.owner && !h.effect.target.evolution && h.effect.ability instanceof PokePower) {
-              h.object=true
+            eff2 = getter (IS_ABILITY_BLOCKED) { Holder h->
```

Comment:
You changed this from a `getterA` to `getter`. Not sure that was intentional.

Reply:
Actually, think you did this with many pokepowers. Is this the way it's supposed to be done on abilities?

Reply:
Yeah, had to register it manually so I can re-check abilities on deactivation of the Poké-Body.

Reply:
> Actually, think you did this with many pokepowers. Is this the way it's supposed to be done on abilities?

Luckily, modern abilities don't get deactivated (most of the time) but entirely removed from the game while the ability is on. However, on that topic GALARIAN_WEEZING from RCL is an example of getters being used this way.